apps/website-example/src/next-wepublish-link.tsx

```diff
@@ -3,11 +3,8 @@ import {BuilderLinkProps} from '@wepublish/website'
 import NextLink from 'next/link'
 
 export const NextWepublishLink = ({children, href, ...props}: BuilderLinkProps) => {
-  const hrefWithoutOrigin =
```

Comment:
Not needed it seems, it doesn't do a full reload.
Would have caused problems in SSR.